app/controllers/froggo_teams_controller.rb

```diff
@@ -15,8 +15,27 @@ def index
 
   def show
     @froggo_team = FroggoTeam.find(params[:id])
-    @froggo_team_members = @froggo_team.github_users
-    @organization_members = @froggo_team.organization.members.all_except(@froggo_team_members)
+    @froggo_team_members = get_froggo_team_members(@froggo_team)
     @github_user = github_user
   end
+
+  private
+
+  def get_froggo_team_members(froggo_team)
+    froggo_team.github_users.map do |member|
+      {
+        id: member.id,
+        avatar_url: member.avatar_url,
+        gh_id: member.gh_id,
+        login: member.login,
+        name: member.name,
+        active: get_member_activation(member, froggo_team),
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Style/TrailingCommaInHashLiteral: Avoid comma after the last item of a hash.